idunn/api/utils.py

```diff
@@ -42,6 +50,68 @@ def fetch_es_poi(id, es) -> dict:
     result['properties'] = properties
     return result
 
+def fetch_bbox_places(bbox, es, indices, categories, max_size) -> list:
+    left, bot, right, top = bbox[0], bbox[1], bbox[2], bbox[3]
+
+    categories = re.findall(r'\((\w*?,\w*?)\)', categories)
+    classes, subclasses, class_sub = [], [], []
+    for pair in categories:
+        if pair.split(",")[1] == "_any":
+            classes.append("class" + pair.split(",")[0])
+            classes.append(pair.split(",")[0])
+        elif pair.split(",")[0] == "_any":
+            subclasses.append("subclass_" + pair.split(",")[1])
+            subclasses.append(pair.split(",")[1])
+        else:
+            class_sub.append("class" + pair.split(",")[0] + " subclass_" + pair.split(",")[1])
+            class_sub.append(pair.split(",")[0] + " " + pair.split(",")[1])
+
+    should_terms = []
+    for term in [classes, subclasses, class_sub]:
```

Comment:
this loop purpose is to avoid any empty list in the queries below, however I have no doubt there are ways with ES to avoid this

Reply:
I think a single list would work here